qcodes_contrib_drivers/drivers/NationalInstruments/RFSG.py

```diff
@@ -0,0 +1,245 @@
+                name_in_library="ConfigureRF",
+                argtypes=[
+                    NamedArgType("vi", ViSession),
+                    NamedArgType("frequency", ViReal64),
+                    NamedArgType("powerLevel", ViReal64),
+                    ]
+                )
+
+        self.add_parameter(name="frequency",
+                           unit="Hz",
+                           get_cmd=partial(self.get_attribute,
+                                           NIRFSG_ATTR_FREQUENCY),
+                           set_cmd=self.set_frequency,
+                           )
+
+        self.add_parameter(name="power_level",
+                           unit="dBm",
+                           label="power level",
+                           get_cmd=partial(self.get_attribute,
+                                           NIRFSG_ATTR_POWER_LEVEL),
+                           set_cmd=self.set_power_level,
+                           )
+
+        self.add_parameter(name="output_on",
+                           label="Output enabled",
+                           get_cmd=partial(self.get_attribute,
+                                           NIRFSG_ATTR_OUTPUT_ENABLED),
+                           set_cmd=partial(self.set_attribute,
+                                           NIRFSG_ATTR_OUTPUT_ENABLED),
+                           val_mapping=on_off_map(on_val=True, off_val=False),
+                           initial_value=False,
+                           )
+
+        self.add_parameter(name="pulse_mod_enabled",
+                           label="Pulse modulation enabled",
+                           get_cmd=partial(self.get_attribute,
+                                           NIRFSG_ATTR_PULSE_MODULATION_ENABLED
+                                           ),
+                           set_cmd=partial(self.set_attribute,
+                                           NIRFSG_ATTR_PULSE_MODULATION_ENABLED
+                                           ),
+                           val_mapping=on_off_map(on_val=True, off_val=False),
+                           initial_value=False,
+                           )
+
+        self.add_parameter(name="clock_source",
+                           label="Reference clock source",
+                           get_cmd=partial(self.get_attribute,
+                                           NIRFSG_ATTR_REF_CLOCK_SOURCE),
+                           set_cmd=self.set_clock_source,
+                           )
+
+        self.initiate()
+        self.connect_message()
+
+    def initiate(self):
+        """
+        Initiate signal generation. This causes the NI-RFSG device to leave
+        the Configuration state.
+        """
+        self.wrapper.Initiate(self._handle)
+
+    def abort(self):
+        """
+        Stop signal generation and return to the Configuration state.
+        """
+        self.wrapper.Abort(self._handle)
+
+    def ConfigureRF(self, frequency: float, power_level: float,
```

Comment:
shouldn't this have a python-style name as per recommendation in the README inside NationalInstruments dir?

Reply:
Fixed in 2abeac7.